changefeedccl: increase chaos roachtest latency limits

samiskin opened the pull request:
Resolves: https://github.com/cockroachdb/cockroach/issues/93238

Since our job-level retry MaxBackoff has increased from 10 seconds to 10 minutes, increase the chaos test latency limits.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/93552)
<!-- Reviewable:end -->

miretskiy commented on pkg/cmd/roachtest/tests/cdc.go:
```diff
@@ -987,8 +987,9 @@ func registerCDC(r registry.Registry) {
-				steadyLatency:      5 * time.Minute,
+				// Higher latencies are permitted due to retry backoff
+				initialScanLatency: 10 * time.Minute,
+				steadyLatency:      30 * time.Minute,
```
would be nice to figure out how to change that default instead of making tests take longer?

samiskin committed:
changefeedccl: increase chaos roachtest latency limits

Resolves: #93238

Since our job-level retry MaxBackoff has increased from 10 seconds to 10 minutes, increase the chaos test latency limits.

Release note: None

<what was there before: Previously, ...>
<why it needed to change: This was inadequate because ...>
<what you did about it: To address this, this patch ...>

miretskiy reviewed (approved):
> Reviewed 3 of 3 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/93552)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @samiskin)

___
*[`pkg/ccl/changefeedccl/retry.go` line 33 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/93552#-NJMKCxHBDcP65L911B0:-NJMKCxHBDcP65L911B1:bt514ht) ([raw file](https://github.com/cockroachdb/cockroach/blob/9af72fb569590f6cd6d80e91ef9956348c709497/pkg/ccl/changefeedccl/retry.go#L33)):*
> ```Go
> 			InitialBackoff: 5 * time.Millisecond,
> 			Multiplier:     2,
> 			MaxBackoff:     250 * time.Minute,
> ```

ouch :) 


<!-- Sent from Reviewable.io -->

samiskin commented:
bors r+

samiskin commented:
blathers backport 22.2
